assets/src/scss/layout/_block-classes.scss

```diff
@@ -1,94 +1,3 @@
-.wp-block-query-pagination {
```

Comment:
Needed to move these styles to the new released listing page's stylesheet.

Reply:
Why aren't you also moving the other styles from this CSS file? It seems they are also related to the listing pages 🤔 
Also, [this `speak` property](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/layout/_block-classes.scss#L101) that we use is currently not supported by any browser, so I'm not sure why it's there 😅

Reply:
It's been moved to [here](https://github.com/greenpeace/planet4-master-theme/pull/1839/files#diff-215e8c1cad357161f9ea21bc668f3da5a9a489a79d302553823b12e1eaf5acf2R198)!

Reply:
> Also, this speak property that we use is currently not supported by any browser, so I'm not sure why it's there 😅

Only [Firefox](https://developer.mozilla.org/en-US/docs/Web/CSS/@counter-style/speak-as#browser_compatibility)

Reply:
That's for the `speak-as` property, not `speak`